src/service/url-replacements-impl.js

```diff
@@ -256,7 +256,8 @@ export class GlobalVariableSource extends VariableSource {
 
         // A temporary work around to extract Client ID from _ga cookie. #5761
         // TODO: replace with "filter" when it's in place. #2198
-        if (scope == '_ga') {
+        const cookieName = opt_cookieName || scope;
```

Comment:
the value transformation will happen for scoped cid when loading from viewer. This won't have a real effect on data but is probably unintended. right? 

`CLIENT_ID(AMP_ECID_GOOGLE,, _ga)` from viewer should not result in a call to `extractClientIdFromGaCookie` but currently it does.  I think it is okay given that this code is temporary anyways..